sharing-data.html

```diff
@@ -261,105 +349,355 @@
+
+<img alt="../images/sharingDataMoveWeb.jpg" class="align-center" src="images/sharingDataMoveWeb.jpg" style="width: 100%;" />
+
+<div class="line-block">
+<div class="line"><br /></div> <!-- end #line -->
+</div> <!-- end #line-block -->
+
+<div class="line-block">
+<div class="line"><br /></div> <!-- end #line -->
+</div> <!-- end #line-block -->
+
+<p class="top_link"><a href="#top"><img src="images/back_to_top.png" alt="Image for link to top of page" style="float: right;"/></a></p>
+
+<h2><a class="anchor" id="default"></a>Changing your default group</h2>
+
+<div class="line-block">
+<div class="line"><br /></div> <!-- end #line -->
+</div> <!-- end #line-block -->
+
+<h3>OMERO.insight</h3>
+
+<ul>
+
+<p>To change your default group, the group that's shown when you log in, go to the <span class="bold">Administration</span> tab.</p>
+
+<p>Select your name in the list of users.</p>
+
+<p>In the <span class="bold">User</span> pane on the right-hand side, select the new default group from the drop-down list.</p>
+
+</ul>
+
+<div class="line-block">
+<div class="line"><br /></div> <!-- end #line -->
+</div> <!-- end #line-block -->
+
+<img alt="../images/sharingDataDefaultInsight.jpg" class="align-center" src="images/sharingDataDefaultInsight.jpg" style="width: 100%;" />
+
+<div class="line-block">
+<div class="line"><br /></div> <!-- end #line -->
+</div> <!-- end #line-block -->
+
+<h3>OMERO.web</h3>
+
+<ul>
+
+<p>To change your default group, the group thats shown when you log in, select <span class="bold">User settings</span> from the drop-down menu in the top right-hand corner.</p>
+
+<p>In the <span class="bold">User</span> tab, select the new default group from the drop-down list.</p>
+
+<p>Click <span class="bold">Save</span> to save your new default group.</p>
+
+</ul>
 
 <div class="line-block">
 <div class="line"><br /></div> <!-- end #line -->
 </div> <!-- end #line-block -->
 
-<img alt="../images/sharingDataMove.jpg" class="align-center" src="images/sharingDataMove.jpg" style="width: 100%;" />
+<img alt="../images/sharingDataDefaultWeb.jpg" class="align-center" src="images/sharingDataDefaultWeb.jpg" style="width: 100%;" />
 
 <div class="line-block">
 <div class="line"><br /></div> <!-- end #line -->
+</div> <!-- end #line-block -->
+
+<div class="line-block">
 <div class="line"><br /></div> <!-- end #line -->
 </div> <!-- end #line-block -->
 
 <p class="top_link"><a href="#top"><img src="images/back_to_top.png" alt="Image for link to top of page" style="float: right;"/></a></p>
 
-<h2><a class="anchor" name="admin" id="admin"></a>Admin Tasks</h2>
+<h2><a class="anchor" id="owner"></a>Managing groups as a Group Owner</h2>
+
+<div class="line-block">
+<div class="line"><br /></div> <!-- end #line -->
+</div> <!-- end #line-block -->
+
+<p>As an owner of a group, you can:</p>
+
+<ul>
+
+<p class="indent1">- manage the membership of that group</p>
+<p class="indent1">- import data on behalf of group members</p>
```

Comment:
I don't think that group owners can Import data for members (and you don't show it below).